Sources/Fuzzilli/Modules/NetworkSync.swift

```diff
@@ -293,7 +301,11 @@ class Connection {
 
 public class NetworkMaster: Module, MessageHandler {
     /// File descriptor of the server socket.
-    private var serverFd: Int32 = -1
+#if os(Windows)
+    private var serverFd: libsocket.socket_t = INVALID_SOCKET
```

Comment:
I'm wondering, can we also define `INVALID_SOCKET` (or some similar constant) on posix as `-1` (from what I can tell, it's guaranteed to be -1 on error), then simplify a lot of this code here and below to always just use `INVALID_SOCKET` instead of having separate code for Windows and POSIX?

Reply:
I definitely prefer that.  I'll locally define `INVALID_SOCKET` to `-1` on non-Windows targets.